lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
+                rule.check(file, errors);
+            } catch (e) {
+                errors.add(e, 1, 0);
```

Comment:
1. Error is an abstract entity about a problem. We have standard notification algorithm (by passing string to errors.add) that notifies user about code style problems. Why we shouldn't use this algorithm to notify user about other troubles? I don't get why `errors.add` is a code style issues collector.

At other hand user shouldn't know by default about problems with code (TypeErrors, etc.). So I prefer to hide it under verbose flag.

I'd like to know what you thinking about current `Errors` object and errors, warnings, etc.
Would be great to know your view at that problem.